asn1/LedgerLedgerLotocol.asn

```diff
@@ -124,8 +124,13 @@ CallSet CALL ::= {
 LedgerLedgerLotocolPacket ::= SEQUENCE {
     -- One byte type ID
     type CALL.&typeId ({CallSet}),
-    -- Allow multiple accounts to be multiplexed on one connection
-    accountId UInt32,
+    -- If the to and/or from addresses can be inferred from the
+    -- context (for example, if the connection is a bilateral
+    -- trustline), these values MAY be set to 0-byte strings
+    -- Peers MAY also use shorter aliases to avoid repeatedly
+    -- sending the full ILP Addresses over the wire
+    to Address,
+    from Address,
```

Comment:
If LLL is not strictly bilateral, we should also think about how to prevent address spoofing. There are two potential attack vectors:
1) A malicious sender fakes the `from` field, so that the recipient thinks the packet was sent by someone else.
2) A malicious receiver manages to receive packets that were indented for another receiver (think of the ILP-equivalent of [ARP spoofing](https://en.wikipedia.org/wiki/ARP_spoofing)).

To prevent such attacks, we need to define how protocol participants authenticate their peers in a multilateral scenario.